lib/phoenix_live_view/channel.ex

```diff
@@ -913,24 +913,31 @@ defmodule Phoenix.LiveView.Channel do
   end
 
   defp render_diff(state, socket, force?) do
-    {socket, diff, components} =
-      if force? or Utils.changed?(socket) do
-        rendered = Utils.to_rendered(socket, socket.view)
-        {socket, diff, components} = Diff.render(socket, rendered, state.components)
+    :telemetry.span([:phoenix, :live_view, :render], %{socket: socket, force?: force?}, fn ->
```

Comment:
Let's only instrument inside the `if` branch. The work outside of the `if` branch is not relevant. :)